pkg/templates/kubeadm/v1beta2/kubeadm.go

```diff
@@ -468,3 +470,11 @@ func kubeProxyConfiguration(s *state.State) *kubeproxyv1alpha1.KubeProxyConfigur
 
 	return kubeProxyConfig
 }
+
+func getContainerLogMaxSize(value string) string {
+	if value == "" {
+		return "100Mi"
+	}
+
+	return value
+}
```

Comment:
Please use defaulting instead of this function.

Reply:
done